include/configuration.h

```diff
@@ -23,8 +24,8 @@ namespace reinforcement_learning { namespace utility {
   class configuration
   {
   public:
-    configuration();
-    ~configuration();
+    VW_DLL_PUBLIC configuration();
```

Comment:
If a user needs to use other fields of this object they need to be public too. Perhaps just move the attribute to the class level?

Reply:
Good point